app/views/planning_applications/publish.html.erb

```diff
@@ -1,9 +1,10 @@
 <%= render "planning_applications/assessment_dashboard" do %>
   <% content_for :title, "Publish the recommendation" %>
 
-  <h2 class="govuk-heading-m">Publish the recommendation</h2>
```

Comment:
I'm not a fan of re-using the "publish" action as a "view decision notice" action. Feels like they should be separate?

What do you think about a new action at `decision_notice_planning_application_path`?

Reply:
Good idea - will be much cleaner 👍